pico/api/src/main/java/io/helidon/pico/api/Interceptor.java

```diff
@@ -50,6 +51,7 @@ public interface Interceptor {
          *
          * @param args the arguments passed
          * @return the result of the call
+         * @throws InvocationException if there is an errors during invocation chain processing
```

Comment:
"there is an error during" or "there are errors during"

Reply:
pushed